PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/forms/CreateArmStorageAccountForm.java

```diff
@@ -220,7 +220,7 @@ protected void doOKAction() {
         // creating from Azure Explorer directly
         setSubscription((SubscriptionDetail) subscriptionComboBox.getSelectedItem());
         if (subscription == null) {
-            final String title = "Creating storage account " + nameTextField.getText() + "...";
+            final String title = "Creating storage account (" + nameTextField.getText() + ")...";
```

Comment:
looks not a good idea. It should be nice if you can move "({0})..." into properties file.